### ISSUE TITLE
Feature request: Add possibility to run in parallel training model

### ISSUE THREAD

**Issue by Pryanic** [LTR4L/ltr4l#304]
I have a powerful computer with many cores, but unfortunately I use only one of them. I think that we can run this algorithms in parallel.

**Comment by Kamulau** [comment]
Yes, that's a great feature! I would also like to get this implemented...

**Comment by Pryanic** [comment]
Other good think it's adding of  possibility training data on GPU
